### PR TITLE
Android sdk

### PR DESCRIPTION
**Pull request by IvanSvirin**

### CODE REVIEW

File: SovrinAgentApp/app/build.gradle [hunk 1]
```diff
@@ -15,12 +15,12 @@ android {
     }
     defaultConfig {
         applicationId "com.luxoft.supplychain.sovrinagentapp"
-        minSdkVersion 26
+        minSdkVersion 21
```
Comment (DevAlexey) [inline SovrinAgentApp/app/build.gradle]: It should be 24?

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
-{"reqSignature":{},"txn":{"data":{"data":{"alias":"Node4","blskey":"2zN3bHM1m4rLz54MJHYSwvqzPchYp8jkHswveCLAEJVcX6Mm1wHQD1SkPYMzUDTZvWvhuE6VNAkK3KxVeEmsanSmvjVkReDeBEMxeDaayjcZjFGPydyey1qxBHmTvAnBKoPydvuTAqx5f7YNNRAdeLmUi99gERUU7TD8KfAa6MpQ9bw","blskey_pop":"RPLagxaR5xdimFzwmzYnz4ZhWtYQEj8iR5ZU53T2gitPCyCHQneUn2Huc4oeLd2B2HzkGnjAff4hWTJT6C7qHYB1Mv2wU5iHHGFWkhnTX9WsEAbunJCV2qcaXScKj4tTfvdDKfLiVuU2av6hbsMztirRze7LvYBkRHV3tGwyCptsrP","client_ip":"52.15.95.244","client_port":9708,"node_ip":"52.15.95.244","node_port":9707,"services":["VALIDATOR"]},"dest":"4PS3EDQ3dW1tci1Bp6543CfuuebjFrg36kLAUcskGfaA"},"metadata":{"from":"TWwCRQRZ2ZHMJFn9TzLp7W"},"type":"0"},"txnMetadata":{"seqNo":4,"txnId":"aa5e817d7cc626170eca175822029339a444eb0ee8f0bd20d3b0b76e566fb008"},"ver":"1"}
-"""
\ No newline at end of file
+const val GENESIS_PATH = "./docker.txn"
+const val TAILS_PATH = "./tails"
+
+//const val nodeIp = """52.15.95.244"""  // amazon
+//const val GENESIS_IP = """52.15.95.244"""  // azure
```
Comment (DevAlexey) [inline SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt]: This is private one

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt [hunk 3]
```diff
@@ -1,22 +1,44 @@
...
+const val GENESIS_IP = """52.224.217.165"""  // azure
+
+
+//const val WS_ENDPOINT = "ws://localhost:8094/ws"
+//const val WS_ENDPOINT = "ws://13.92.115.239:8094/ws"  // azure
+//const val WS_ENDPOINT = "ws://3.17.65.252:8094/ws"  // amazon
+const val WS_ENDPOINT = "ws://52.224.104.135:8094/ws"  // client
```
Comment (DevAlexey) [inline SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt]: Should not have any hardcode, this is SDK right?

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/data/VerificationEvent.kt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+data class VerifierInfo(
+    val did: String,
+    val name: String,
+    val address: String,
+    val contactPhone: String
+)
+
+fun verifierInfoFromDid(did: String) =
+    VerifierInfo(did,
+        name = "Cherry Hill Medical Center",
```
Comment (DevAlexey) [inline SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/data/VerificationEvent.kt]: Project specific references?

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/domain/di/Modules.kt [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+    }
+}
+
+val agentConnection = PythonRefAgentConnection()
+val agentConnectionProgress = agentConnection.connect(WS_ENDPOINT, WS_LOGIN, WS_PASS).toBlocking().value()
+
+
+//Async indy initialization for smooth UX
+//lateinit var pool: Pool
+//lateinit var wallet: Wallet
```
Comment (DevAlexey) [inline SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/domain/di/Modules.kt]: What is this?

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/views/activities/MainActivity.kt [hunk 6]
```diff
@@ -75,17 +80,22 @@ class MainActivity : AppCompatActivity() {
...
-                .findAllAsync()  // todo: replace with `findFirstAsync` after you fix realm.delete on each update
+                .findAllAsync()
 
         setSupportActionBar(toolbar)
         supportActionBar?.setDisplayHomeAsUpEnabled(false)
 
-        nameClaims.addChangeListener { claims ->
-            val userName = claims.first()?.value ?: ""
-            supportActionBar?.title = userName
+//        nameClaims.addChangeListener { claims ->
```
Comment (DevAlexey) [inline SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/views/activities/MainActivity.kt]: Remove?

File: SovrinAgentApp/app/build.gradle (hunk 1 above)
Comment (voddan) [reply SovrinAgentApp/app/build.gradle]: Yes I think we agreed on 24 because it unlocks a bunch of new APIs

Comment (IvanSvirin) [reply SovrinAgentApp/app/build.gradle]: ok

Comment (IvanSvirin) [reply SovrinAgentApp/app/build.gradle]: changed

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt (hunk 2 above)
Comment (IvanSvirin) [reply SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt]: deleted

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt (hunk 3 above)
Comment (IvanSvirin) [reply SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/application/Const.kt]: deletd

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/data/VerificationEvent.kt (hunk 4 above)
Comment (IvanSvirin) [reply SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/data/VerificationEvent.kt]: cleared

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/domain/di/Modules.kt (hunk 5 above)
Comment (IvanSvirin) [reply SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/domain/di/Modules.kt]: deleted

File: SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/views/activities/MainActivity.kt (hunk 6 above)
Comment (IvanSvirin) [reply SovrinAgentApp/app/src/main/java/com/luxoft/supplychain/sovrinagentapp/views/activities/MainActivity.kt]: deleted

**Pull request closed without merging**